pkg/config/setup/config.go

```diff
@@ -374,7 +374,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	// 	- `statefulsets.apps`
 	// 	- `pods`
 	// 	- `nodes`
-	config.BindEnvAndSetDefault("kubernetes_resources_labels_as_tags", map[string]map[string]string{})
+	config.BindEnvAndSetDefault("kubernetes_resources_labels_as_tags", "{}")
```

Comment:
Same here.

Reply:
Discussed on slack, and agreed that using `"{}"` is a valid usecase.